src/NHibernate.Test/NHSpecificTest/NH1280/NH1280Fixture.cs

```diff
@@ -41,9 +41,6 @@ protected override void OnTearDown()
 		[Test]
 		public void HavingUsingSqlFunctions_Concat()
 		{
-			if (TestDialect.HasBrokenTypeInferenceOnSelectedParameters)
```

Comment:
This test does not have anything to do with selected parameters. The only dialect that currently has `HasBrokenTypeInferenceOnSelectedParameters == true` is `SapSQLAnywhere17Dialect`. This test tests concat, so I thought this change should re-enable this test.